app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,233 @@
+h1 {
+    .lib-visually-hidden();
+}
+.section.section__slider {
+    padding-bottom: 30px;
+}
+
+.section {
+    padding-bottom: 30px;
+    h2,
+    h3 {
+        font-size: 24px;
```

Comment:
Please use `.lib-font-size()` mixin, check it here:
`/lib/web/css/source/lib/_utilities.less : line 53`